articles/active-directory-b2c/custom-policies-series-sign-up-or-sign-in-federation.md

```diff
@@ -517,6 +517,7 @@ Use the following steps to add a combined local and social account:
     ```xml
         <OutputClaim ClaimTypeReferenceId="authenticationSource" DefaultValue="localIdpAuthentication" AlwaysUseDefaultValue="true" />
     ```
+    The same code should be added to the `UserSignInCollector` self-asserted technical profile. Otherwise, a user encounters the `AADB2C90037` error during local sign-in at orchestration step #6, since the `authenticationSource` claim is not assigned any value and the `alternativeSecurityId` claim is not found.
```

Comment:
@esinmy This one is not clear. Can you provide a snippet to how the 'same code' should be added to the `UserSignInCollector` self-asserted technical profile. Which code snippet do you refer to as _The same code_?

Reply:
Hi @kengaderdus
By _The same code_, I meant this line:
```xml
<OutputClaim ClaimTypeReferenceId="authenticationSource" DefaultValue="localIdpAuthentication" AlwaysUseDefaultValue="true" />
```
The `UserSignInCollector` technical profile should be:
```xml
<TechnicalProfile Id="UserSignInCollector">
    <DisplayName>Local Account Signin</DisplayName>
    <Protocol Name="Proprietary"
        Handler="Web.TPEngine.Providers.SelfAssertedAttributeProvider, Web.TPEngine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" />
    <Metadata>
        <Item Key="setting.operatingMode">Email</Item>
        <Item Key="SignUpTarget">AccountTypeInputCollectorClaimsExchange</Item>
    </Metadata>
    <DisplayClaims>
        <DisplayClaim ClaimTypeReferenceId="email" Required="true" />
        <DisplayClaim ClaimTypeReferenceId="password" Required="true" />
    </DisplayClaims>
    <OutputClaims>
        <OutputClaim ClaimTypeReferenceId="email" />
        <OutputClaim ClaimTypeReferenceId="password"  />
        <OutputClaim ClaimTypeReferenceId="objectId" />
        <OutputClaim ClaimTypeReferenceId="authenticationSource" DefaultValue="localIdpAuthentication" AlwaysUseDefaultValue="true" />
    </OutputClaims>
    <ValidationTechnicalProfiles>
        <ValidationTechnicalProfile ReferenceId="SignInUser" />
    </ValidationTechnicalProfiles>
</TechnicalProfile>
```